core_contracts/governance/governance.py

```diff
@@ -224,27 +304,37 @@ def totalBaln(self, _day: int) -> int:
         return total_baln_token
 
     @external
-    def executeVoteAction(self, vote_index: int) -> None:
+    def evaluateVote(self, name: str) -> None:
```

Comment:
In the cases where we have the vote name as a parameter let's change it to the vote index. I had originally thought that using the name would make it more human readable, but that is not necessary and now we have some using the index and some using the name. My mistake to have used the name in the first place.

Reply:
Yeah, I was not sure which one you preferred. Like you, I also thought it would be more readable with the name. Making the changes.